src/test/collaterals.sol

```diff
@@ -0,0 +1,1513 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.6.12;
+
+contract Collaterals {
+    struct SystemValues {
```

Comment:
I think we could keep `SystemValues` struct in base tests in L64 as not used n `collaterals.sol`, following a similar pattern as per `SpellValues`.